src/components/node/sul-library/page-display.tsx

```diff
@@ -16,9 +18,8 @@ const SulLibrary = async ({node, ...props}: {node: NodeSulLibrary}) => {
     timeZone: "America/Los_Angeles",
   })
 
-  return (
-    <article className="mb-50 @container" {...props}>
-      <SulLibraryMetadata node={node} />
+  const content = (
```

Comment:
Don't use variables for components. Just render the parts in the component below.